ci-operator/templates/openshift/longlived/longlived-test.yaml

```diff
@@ -0,0 +1,156 @@
+kind: Template
```

Comment:
The directory and file naming feels weird.  But is fine for now

Reply:
i renamed it to endurance. i'm sure it's still weird but at it's consistent anyway.